server/sonar-server/src/test/java/org/sonar/server/issue/ws/AssignActionTest.java

```diff
@@ -156,8 +153,9 @@ public void unassign_with_empty_assignee_param() throws Exception {
 
   @Test
   public void nothing_to_do_when_new_assignee_is_same_as_old_one() throws Exception {
-    db.users().insertUser("arthur");
     IssueDto issue = newIssueWithBrowsePermission();
+    UserDto userDto = db.users().insertUser(PREVIOUS_ASSIGNEE);
```

Comment:
I wouldn't call it magic. It's closer to the default behavior for on premise instances